tests/test_modal.ts

```diff
@@ -0,0 +1,87 @@
+import Modal from '@/components/modal.vue';
+import { mount } from '@vue/test-utils';
+
+describe('Modal.vue', () => {
+    test('Modal open, close, and toggle methods open and close modal', () => {
```

Comment:
Can you add a test/augment this test to make sure that the modal stuff is only visible with is_open is true? You should be able to use https://vue-test-utils.vuejs.org/api/wrapper/#isvisible to check that